packages/frontend/src/page/components/answer-form/answer/index.tsx

```diff
@@ -16,68 +13,80 @@ import {
     isQuestionNumerical,
 } from "../../../../utils";
 import { RealityQuestion } from "../../../types";
+import { LearnMore } from "../../learn-more";
 
 interface AnswerProps {
     t: NamespacedTranslateFunction;
     question: RealityQuestion;
 }
 
 export const Answer = ({ t, question }: AnswerProps): ReactElement => {
-    const nativeCurrency = useNativeCurrency();
-
     if (isQuestionAnswerMissing(question))
         return (
             <Typography variant="lg">{t("label.answer.missing")}</Typography>
         );
 
     const finalized = isQuestionFinalized(question);
-    const formattedBond = formatUnits(question.bond, nativeCurrency.decimals);
+    // TODO: display the bond somewhere
+    // const formattedBond = formatUnits(question.bond, nativeCurrency.decimals);
     return (
         <div className="flex flex-col justify-between gap-3">
-            {isQuestionBoolean(question) && (
-                <Typography variant="lg">
-                    {t("label.answer.answer", {
-                        answer:
-                            question.bestAnswer === BYTES32_ZERO
-                                ? t("label.answer.no")
-                                : t("label.answer.yes"),
-                        bond: formattedBond,
-                        symbol: nativeCurrency.symbol,
-                    })}
+            {finalized && (
+                <Typography
+                    variant="h5"
+                    weight="bold"
+                    className={{ root: "mb-4" }}
+                >
+                    {t("label.answer.finalized")}
                 </Typography>
             )}
+            {isQuestionBoolean(question) && (
+                <div className="flex flex-col gap-1">
+                    <Typography>
+                        {t("label.answer.answer", {
+                            answer:
+                                question.bestAnswer === BYTES32_ZERO
```

Comment:
In this case if the question is of boolean type and the answer is invalid we end up in this branch and show "yes" as an answer, no? If the `isQuestionBoolean` function also checks for answer validity, let's make it clear in the name, otherwise it's a bit sketchy.

Reply:
Yes, the `isQuestionBoolean` checks also that the answer is not invalid nor answered too soon. I guess we can rename it to something like `isQuestionAnswerPurelyBoolean`? The same goes for the numerical case.